app/views/comments/_comment_date.erb

```diff
@@ -1,7 +1,15 @@
 <div class="comment-date">
   <a href="<%= decorated_comment.path %>">
+  <span class="hidden m:inline-block">
     <time datetime="<%= decorated_comment.published_timestamp %>">
       <%= decorated_comment.readable_publish_date %>
     </time>
+    </span>
   </a>
-</div>
+  <% if !decorated_comment.edited_at? %>
+    •
+    <span title="<%= decorated_comment.edited_at.strftime('%A, %d %B %Y, %T') %>">
+      EDITED
```

Comment:
This needs to be changed to Edited for <https://github.com/forem/forem/pull/10524#issuecomment-702664568>